abstractions/go/serialization/time_only.go

```diff
@@ -0,0 +1,62 @@
+package serialization
+
+import (
+	"errors"
+	"strings"
+	"time"
+)
+
+// TimeOnly is represents the time part of a date time (time) value.
+type TimeOnly struct {
+	time time.Time
+}
+
+var timeOnlyFormat = "15:04:05.000000000"
```

Comment:
Should `timeOnlyFormat` and `timeOnlyParsingFormats` be constants as well?

Reply:
```suggestion
const timeOnlyFormat = "15:04:05.000000000"
```

Reply:
maps can't be constants in Go.